### DISCUSSION TITLE
In http endpoint -> json -> Is there a way to include information whether an attachment was an inline attachment?

### DISCUSSION THREAD

**Discussion by rakurtz** [postalserver/postal#3003] [Help with using Postal]
Hey there,

i am using postal server for my just founded company. Every email sent to an address on postal is forwarded by postal as a json to http endpoint. I very much like the way, the information of the email is structured into the json. Unfortunately there is no way for me to tell if any of the attachments listed inside the json was an **inline attachment**. 

All i get right now is
```
"attachments": [
        {
            "filename": "image0.jpeg",
            "content_type": "image/jpeg",
            "size": 122509,
            "data": "/9j/4AAQSkZJRgABAQAASABIAAD/4QBYRXhpZgAATU0AKgAAAAgAAgESAAMA and so on..."
        }
    ]
```

Are there any ideas on how to preserve or retrieve this information?

Thank's a lot!

**Comment by willpower232** [comment]
If they don't appear under the attachments tab in the UI then Postal doesn't know they exist so I think this is not straightforward currently.

See also #1963 (I thought it had been discussed more recently but can't remember where off the top of my head)

**Reply by rakurtz** [reply]
Hey Will, thanks for your quick reply. I maybe I didn't explain my question clear enough. The inline attachment is listed in the json file's attachment array perfectly. It's only the information about a specific attachment was sent as an inline attachment, that would be great to preserve in the attachments array's information (**content disposition**).

Something like this:

```
"attachments": [
        {
            "filename": "image0.jpeg",
            "content_type": "image/jpeg",
            "content_disposition": "inline",          # or "attachment"
            "size": 122509,
            "data": "/9j/4AAQSkZJRgABAQAASABIAAD/4QBYRXhpZgAATU0AKgAAAAgAAgESAAMA and so on..."
        }
    ]
```
